.gitignore

```diff
@@ -22,3 +22,7 @@
 .vscode/*
 code/software/lcd-ili9341/images/BMPToRLE555
 code/software/lcd-ili9341/rle_images.h
+code/firmware/emucon/emucon2.tos
```

Comment:
These gitignore changes don't feel generally applicable, can they be removed please?